index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Raj Portfolio</title>
+</head>
+ <body>
+  <main class="rj">
+   <header>
+      <a href="#"class="logo">My Logo</a>
+      <nav class="nav">
+          <div class="button"><a href="#"><img src="image/Union.png" alt="button"></a></div>
+      </nav>
+  </header>
+  <section>
+    <h1>Hey There. I'm Vivek Raj glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p2>
+
+          <h5 class="connect">LET'S CONNECT</h5>
+          <ul class="icon">
+              <li><a href="https://twitter.com/BossVivekRaj"><img src="image/twitter.png" alt="twitter-icon"></a></li>
+              <li><a href="https://www.linkedin.com/in/vivek-raj-a78b92256/"><img src="image/Linkedin icon.png" alt="linkedin-icon"></a></li>
+              <li><a href="https://medium.com/"><img src="image/medium.png" alt="medium-icon"></a></li>
+              <li><a href="https://github.com/VivekRaaZ"><img src="image/github.png" alt="github-icon"></a></li>
+              <li><a href="https://www.angellist.com/"><img src="image/angellist.png" alt="angellist-icon"></a></li>
+          </ul>
+  </section>
```

Comment:
After resolving the above-suggested issues, kindly readjust the background image, The comparison is given below:
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219883899-d665fb0b-c1fb-4bb3-b6f2-d0e5147f5963.png)| ![image](https://user-images.githubusercontent.com/61361037/219883878-f7a81df8-df8c-45bf-acc2-c6790732eafa.png)|